src/main/java/org/aesh/readline/ShellImpl.java

```diff
@@ -69,6 +159,8 @@ public String readLine() throws InterruptedException {
 
     @Override
     public String readLine(Prompt prompt) throws InterruptedException {
```

Comment:
should we create a readline(Prompt, boolean) method as well?

Reply:
Prompting doesn't require pausing. When prompting the user, any collected data needs first to be printed then prompt is displayed. So I would say that we don't need a new method.

Reply:
ah, true. thanks!